src/app/components/game-center/game-center.component.ts

```diff
@@ -3,21 +3,21 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-game-center',
   templateUrl: './game-center.component.html',
-  styleUrls: ['./game-center.component.scss']
+  styleUrls: ['./game-center.component.scss'],
 })
 export class GameCenterComponent {
-
   word = '';
+  lenghtWord = this.word.length < 5;
```

Comment:
```suggestion
  isMaxLengthWord(): boolean {
     return this.word.length < 5;
  }
```

Si quieres guardar un booleano para comprobar el rango máximo, mejor una función, y te ahorras esa asignación fea tanto en la declaración, como en el default del switch